assets/sass/modules/_modal.scss

```diff
@@ -21,6 +21,12 @@
             font-weight: 900;
             color: $highlight;
         }
+        .greeting-buttons {
+            display: flex;
+            justify-content: center;
+            margin-top: 10px;
+        }
+
```

Comment:
the icons are not centered properly 

![image](https://user-images.githubusercontent.com/60546840/105910312-9bb9fa80-5fdd-11eb-82eb-24eb79b25d52.png)

Reply:
give those button tags some class name like `modal-buttons`

then add some padding in those button
```css
.visual-button.modal-buttons {
    padding-top: 4px;
}
```